client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/FamilyMembers/FamilyMembersTable/FamilyMembersTable.tsx

```diff
@@ -0,0 +1,119 @@
+
+import useStyles from './FamilyMembersTableStyles';
+import FamilyContactsTableHeadersNames, { FamilyContactsTableHeaders } from '../../../../../FamilyContactsDialog/FamilyContactsTable/FamilyContactsTableHeaders';
+
+const birthDateFormat = 'dd/MM/yyyy';
+
+const FamilyMembersTable: React.FC<Props> = (props: Props) => {
+    
+    const { familyMembers } = props;
+    const classes = useStyles();
+
+    const [selectedFamilyMembers, setSelectedFamilyMembers] = useState<InvolvedContact[]>([]);
+
+    const selectRow = (selectedFamilyMember: InvolvedContact) => {
+        const familyMemberIndex = selectedFamilyMembers.findIndex(checkedRow => selectedFamilyMember === checkedRow);
+        if (familyMemberIndex !== -1) {
+            setSelectedFamilyMembers(selectedFamilyMembers.filter(member => member !== selectedFamilyMember));
+            selectedFamilyMember.selected = false;
+        } else {
+            setSelectedFamilyMembers([...selectedFamilyMembers, selectedFamilyMember]);
+            selectedFamilyMember.selected = true;
+        }
+    }
+    
+    useEffect(() => {
+        familyMembers.map((familyMember: InvolvedContact) => {
+            familyMember.selected = false;
+        });
+    }, []);
+
+    const counterDescription: string = useMemo(() => {
+        return selectedFamilyMembers.length > 0 ?
+        selectedFamilyMembers.length === 1 ?
+                'נבחר מגע משפחה אחד' :
+                'בסה"כ נבחרו ' + selectedFamilyMembers.length + ' מגעי משפחה'
+            : ''
+    }, [selectedFamilyMembers]);
+
+    const isRowSelected = (selectedFamilyMember: InvolvedContact) => selectedFamilyMembers?.includes(selectedFamilyMember);
+
+    const convertToIndexedRow = (row: InvolvedContact) : IndexedContactRow => ({
+        [FamilyContactsTableHeadersNames.FAMILY_RELATIONSHIP]: row.familyRelationship,
+        [FamilyContactsTableHeadersNames.FIRST_NAME]: row.firstName,
+        [FamilyContactsTableHeadersNames.LAST_NAME]: row.lastName,
+        [FamilyContactsTableHeadersNames.IDENTIFICATION_TYPE]: row.identificationType,
+        [FamilyContactsTableHeadersNames.IDENTIFICATION_NUMBER]: row.identificationNumber,
+        [FamilyContactsTableHeadersNames.BIRTH_DATE]: row.birthDate,
+        [FamilyContactsTableHeadersNames.PHONE_NUMBER]: row.phoneNumber,
+        [FamilyContactsTableHeadersNames.ADDITIONAL_PHONE_NUMBER]: row.additionalPhoneNumber,
+        [FamilyContactsTableHeadersNames.ISOLATION_CITY]: row.isolationCity
+    });
+
+    const getTableCell = (row: IndexedContactRow, cellName: string) => {
+        const cellValue = row[cellName as FamilyContactsTableHeadersNames];
+        if (!cellValue) return '---'; 
+        switch (cellName) {
+            case FamilyContactsTableHeadersNames.BIRTH_DATE: {
+                return format(new Date(cellValue), birthDateFormat);
+            }
+            default: 
+                return cellValue;
+        }
+    }
+
```

Comment:
move the functions that are used in both tables to a hook, and move the headers to the same directory